src/App.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import Home from "../src/pages/Home/Home";
+import Login from "../src/pages/Login/Login";
+import NotFoundPage from "../src/pages/NotFoundPage/NotFoundPage";
+
+const App = () => {
+  return (
+    <Router>
+      <Route path="/" exact component={Home} />
+      <Route path="/login" component={Login} />
```

Comment:
Can you add an `exact` prop here too. It seems without it it would also render the `/` path.

https://medium.com/@sampath.katari/use-of-exact-prop-for-route-in-react-router-v4-fdbe20e8925d